api/v0/observe/observe.proto

```diff
@@ -52,7 +52,23 @@ service ObserveService {
   rpc GetPosixSignalsStream(GetPosixSignalsStreamRequest) returns (stream GetPosixSignalsStreamResponse) {}
 }
 
-message GetPosixSignalsStreamRequest { 
+/// Request a stream of POSIX signals
+message GetPosixSignalsStreamRequest {
+  /// The workload to which te response will be scoped. If no workload is
+  /// specified, a stream of all POSIX signals on the host will be returned.
+  Workload workload = 1;
+}
+
+enum WorkloadType {
+  WORKLOAD_TYPE_UNSPECIFIED = 0;
+  WORKLOAD_TYPE_CELL = 1;
+  WORKLOAD_TYPE_CONTAINER = 2;
```

Comment:
This should be a PodSandbox and not a container to keep with the "service level" types.

Also we will likely want a container type in the future, but we can cross that bridge later.